src/Slider.jsx

```diff
@@ -80,32 +68,33 @@ const propTypes = {
   snapPoints: PropTypeArrOfNumber,
   // the values
   values: PropTypeArrOfNumber,
+  // react-with-styles export
+  styles: PropTypes.object,
```

Comment:
These test utils might be useful: https://github.com/Khan/aphrodite/blob/61d1ec7062f451cf8c184af8dfb948d6a7708dd5/src/exports.js#L45-L73